bagua/torch_api/communication.py

```diff
@@ -49,6 +49,8 @@
 # Process group count for default naming
 _group_count = 0
 
+# Torch process group to bagua process group
+_torch_to_bagua_pg_map = {}
```

Comment:
What is the process group is destroyed? It seems that they will never be released in current implementation

Reply:
TorchProcessGroup destory has not be handle yet, I have no good way deal with it.

I try to patch bagua pg on TorchProcessGroup, but if patch class is NCCLProcessGroup, it not work. because C object (NCCLProcessGroup) not support add attribute. The final plan is as it is now. Any suggestions ?